tap_google_sheets/client.py

```diff
@@ -145,6 +146,14 @@ def __init__(self,
         self.__expires = None
         self.__session = requests.Session()
         self.base_url = None
+
+        # if supports_all_drives is given as invalid value then raise exception
+        if supports_all_drives not in ('true', 'True', True, 'false', 'False', False, None):
+            raise Exception("Invalid Parameter value: The given value of the supports_all_drives is an invalid value.")
```

Comment:
```suggestion
```

It feels like a waste of time check the values like this. Incorrect values should just map to false and then we move on. 

Assuming someone reads this and disagrees with me, I want to argue at the very least `"True"` and `"False"` should not to be allowed to be valid values. The config file should be JSON; so this field `support_all_drives` feels like a boolean value. 

`"True"` is just a random JSON string. It's not a JSON boolean. It's not a Python boolean. It's a mix of Python and JSON and just a tiny quality of life feature for someone that runs the tap themselves and didn't read the README.

I think instead of throwing an exception, we just note in the README what the valid values are and hope that people read the docs

Reply:
@luandy64, updated code as per your suggestion.